src/routes/users.rs

```diff
@@ -50,6 +84,38 @@ pub async fn user_get(
     }
 }
 
+#[get("{id}/mods")]
+pub async fn mods_list(
```

Comment:
A name like `user_mods_list` may be better for this, or even just change the `{id}` to `{user_id}` to make what this route does more obvious.